examples/multiples/os.go

```diff
@@ -0,0 +1,49 @@
+//go:build !baremetal
+
+package main
+
+import (
+	"context"
+	"errors"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+)
+
+func initExitHandler() context.Context {
+	return contextWithSignal(context.Background())
+}
+
+// ContextWithSignal creates a context canceled when SIGINT or SIGTERM are notified
+func contextWithSignal(ctx context.Context) context.Context {
+	newCtx, cancel := context.WithCancel(ctx)
+	signals := make(chan os.Signal)
```

Comment:
```suggestion
	signals := make(chan os.Signal, 1)
```
My editor warned me the channel shall be buffered here.